tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -182,6 +189,16 @@ def logcat_cmd(self):
         cmd.extend(['logcat', '*:D'])
         return cmd
 
+    def maybe_parse_tombstone(self, logger):
+        if self.stackparser_script:
+            cmd = [self.stackparser_script, "-a", "-w"]
+            if self.device_serial:
+                cmd.extend(["--device", self.device_serial])
+            cmd.extend(["--output-directory", self.output_directory])
+            raw_output = subprocess.check_output(cmd)
+            for line in raw_output.splitlines():
+                logger.process_output("TRACE", line, "logcat")
```

Comment:
If we actually find a crash can we call `logger.log_crash` so it is logged as a crash?

Reply:
So I would say this is similar to the discussion for check_crash at above, and let's don't do this yet.